jgrapht-core/src/main/java/org/jgrapht/alg/PathValidator.java

```diff
@@ -27,8 +27,9 @@
  * 
  * @author Assaf Mizrachi
  * @since July, 21, 2016
- *
+ * @deprecated Use {@link org.jgrapht.alg.shortestpath.PathValidator} instead.
```

Comment:
Indeed. I have not done so, in order to avoid a need of rebasing in the other pull-request. 
 - If the other is merged first, I will rebase this one. 
 - Otherwise, I will submit a new pull-request after the other is merged.